v3/doio/kb30/kb30-01.json

```diff
@@ -0,0 +1,165 @@
+{
+  "name": "DOIO",
+  "vendorId": "0xD010",
+  "productId": "0x3001",
+  "keycodes": ["qmk_lighting"],
+    "menus": [
+    {
+      "label": "Lighting",
+      "content": [
+        {
+          "label": "Backlight",
+          "content": [
+            {
+              "label": "Brightness",
+              "type": "range",
+              "options": [0, 255],
+              "content": ["id_qmk_rgb_matrix_brightness", 3, 1]
+            },
+            {
+              "label": "Effect",
+              "type": "dropdown",
+              "content": ["id_qmk_rgb_matrix_effect", 3, 2],
+              "options": [
+                "All Off",
+                "Solid Color",
+                "Alphas Mods",
+                "Gradient Up/Down",
+                "Gradient Left/Right",
+                "Breathing",
+                "Band Sat.",
+                "Band Val.",
+                "Pinwheel Sat.",
+                "Pinwheel Val.",
+                "Spiral Sat.",
+                "Spiral Val.",
+                "Cycle All",
+                "Cycle Left/Right",
+                "Cycle Up/Down",
+                "Rainbow Moving Chevron",
+                "Cycle Out/In",
+                "Cycle Out/In Dual",
+                "Cycle Pinwheel",
+                "Cycle Spiral",
+                "Dual Beacon",
+                "Rainbow Beacon",
+                "Rainbow Pinwheels",
+                "Raindrops",
+                "Jellybean Raindrops",
+                "Hue Breathing",
+                "Hue Pendulum",
+                "Hue Wave",
+                "Pixel Rain",
+                "Pixel Flow",
+                "Pixel Fractal",
+                "Typing Heatmap",
+                "Digital Rain",
+                "Solid Reactive Simple",
+                "Solid Reactive",
+                "Solid Reactive Wide",
+                "Solid Reactive Multi Wide",
+                "Solid Reactive Cross",
+                "Solid Reactive Multi Cross",
+                "Solid Reactive Nexus",
+                "Solid Reactive Multi Nexus",
+                "Spash",
+                "Multi Splash",
+                "Solid Splash",
+                "Solid Multi Splash"
```

Comment:
![image](https://github.com/the-via/keyboards/assets/17491233/4a2617dc-5cf9-4b45-a5da-939c4fc94091)

You've enabled all of them, but your VIA does not match QMK firmware. You should delete the ones in the VIA json list that isnt enabled in QMK

Reply:
@Xelus22  Hi, thank you for your help,  maybe this time is right.